extras/test/src/util/PropertyTestUtil.cpp

```diff
@@ -16,3 +18,13 @@ unsigned long getTime()
 {
   return 0;
 }
+
+unsigned long TimeService::getLocalTime()
+{
+  return getTime();
+}
+
+TimeService* ArduinoIoTCloudTimeService() {
```

Comment:
This looks somewhat like you want to use a `Singleton` (also see comment below). In general I'd advise to avoid singletons but if you must have them then please follow this pattern:
```C++
/* Singleton.h */
class Singleton
{
public:
  Singleton(Singleton &) = delete;
  void operator = (Singleton &) = delete;

  static Singleton & instance();
  static void destroy();
private:
  static Singleton * _p_instance;
};
/* Singleton.cpp */
Singleton * Singleton::_p_instance{nullptr};
Singleton & Singleton::instance()
{
  if (!_p_instance) {
    _p_instance = new Singleton();
  }
  return *_p_instance;
}

void Singleton::destroy()
{
  delete _p_instance;
  _p_instance = nullptr;
}
```

Reply:
implemented with the pattern above using references -> see https://github.com/arduino-libraries/ArduinoIoTCloud/commit/920e920a7f4125e6272b8c6ba814301eb9f0442f